tests/Detector.Tests/Hugo/HugoDetectorTest.cs

```diff
@@ -63,7 +64,7 @@ public void IsHugoApp_ReturnsTrue_ForAllSupportedConfigurationKeys_InTomlFile(st
         {
             // Arrange
             var appDir = CreateAppDir();
-            WriteFile($"{configurationKeyName}=\"test\"", appDir, HugoConstants.TomlFileName);
+            WriteFile($"{configurationKeyName}=\"test\"", appDir, HugoConstants.TomlFileNames.First());
```

Comment:
We should extract out the name of the file to be an additional argument of this test to ensure that the support configuration keys work for all versions of `.toml` file.

This comment also applies for the other tests below where we've made the same change.